### PR TITLE
added loop to keep car alive while rsync is running

### PR DESCRIPTION
**Pull request by pgardiner**
I added the tesla_api.py wake up command to work with rsync. It calls the same script that coredumperror added last week for cifs. Because rsync runs for a long time without returning, I run a loop in the background to send a wake up every two minutes. It stops once rsync is no longer running (although it would likely die anyways once the main script stops).

### CODE REVIEW

File: run/rsync_archive/archive-clips.sh [hunk 1]
```diff
@@ -1,5 +1,27 @@
 #!/bin/bash -eu
 
+# Call wake_up_vehicle Tesla API every 120 seconds
+# as long as rsync is running. This has to be run
+# in the background.  
+function keepVehicleAwake() {
+  if [ -f /root/bin/tesla_api.py ]
+    then
```
Comment (marcone) [inline run/rsync_archive/archive-clips.sh:8]: coding style in other scripts is to indent "then" to the same level as the "if"

**Review by marcone — COMMENTED** [review]
I have a minor concern that the keepVehicleAwake function will still be running after archive-clips exits. I wonder if there's any way to gracefully stop it (i.e. don't just kill it).

Also, it seems it would be better to move this functionality to the archiveloop script (and remove it from the CIFS archive-clips.sh) so that it works with every backup method.

File: run/rsync_archive/archive-clips.sh (hunk 1 above)
Comment (pgardiner) [reply run/rsync_archive/archive-clips.sh:8]: Makes sense. I have rewritten it to be more generic, and then kill the PID after archive-clips.sh returns. I can submit another pull request, but I can't test the CIFS archive-clips.sh. I have to test a few times first though.

Comment (pgardiner) [reply run/rsync_archive/archive-clips.sh:8]: It's working so far, but need to do a bit more testing. here is the change:
https://github.com/pgardiner/teslausb/commit/fdae20a006091699c1966e42772d74a0e9c5055c

**Comment by pgardiner** [comment]
moving functionality into archiveloop

**Pull request closed without merging**